### PR TITLE
Update Nextcloud docs screenshots after NC 2.6.2 release

### PR DESCRIPTION
**Pull request by MayaBerd**
Update Nextcloud docs screenshots after NC 2.6.2 release

### CODE REVIEW

File: docs/user-guide/file-management/nextcloud-integration/NC_1_00-FileWPRelation.png
Comment (dominic-braeunlein) [inline docs/user-guide/file-management/nextcloud-integration/NC_1_00-FileWPRelation.png:1]: Hi Maya, I think you accidentally uploaded the same screenshot again.

File: docs/user-guide/file-management/nextcloud-integration/NC_1_01-FileWPActions.png
Comment (dominic-braeunlein) [inline docs/user-guide/file-management/nextcloud-integration/NC_1_01-FileWPActions.png:1]: This one too

**Review by dominic-braeunlein — CHANGES_REQUESTED** [review]
I think something went wrong with 2 of the 3 screenshots

File: docs/user-guide/file-management/nextcloud-integration/NC_1_00-FileWPRelation.png
Comment (MayaBerd) [reply docs/user-guide/file-management/nextcloud-integration/NC_1_00-FileWPRelation.png:1]: I've adjusted the screenshot, but saved it under same name. The change on screen was minimal (avatar alignment more the left), so it looks almost the same.

File: docs/user-guide/file-management/nextcloud-integration/NC_1_01-FileWPActions.png
Comment (MayaBerd) [reply docs/user-guide/file-management/nextcloud-integration/NC_1_01-FileWPActions.png:1]: same with this screenshot

**Review by dominic-braeunlein — APPROVED** [review]
LGTM 👍

**Pull request merged**